.github/pull_request_template.md

```diff
@@ -16,18 +16,21 @@ Any User Interface Changes (namelist or namelist defaults changes)?
  - [ ] No
 
 Testing performed if application target is CESM:(either UFS-S2S or CESM testing is required):
-- [ ] (required) CIME_DRIVER=nuopc scripts_regression_tests.py
+- [ ] (recommended) CIME_DRIVER=nuopc scripts_regression_tests.py
    - machines:
    - details (e.g. failed tests):
-- [ ] (required) CESM testlist_drv.xml
+- [ ] (recommended) CESM testlist_drv.xml
    - machines and compilers:
    - details (e.g. failed tests):
 - [ ] (optional) CESM prealpha test
    - machines and compilers
    - details (e.g. failed tests):
+- [ ] (other) please described in detail
+   - machines and compilers
+   - details (e.g. failed tests):
 
 Testing performed if application target is UFS-S2S:
-- [ ] (required) UFS-S2S testing
+- [ ] (recommended) UFS-S2S testing
    - description:
    - details (e.g. failed tests):
 
```

Comment:
Is it possible to add UFS-HAFS to the list?

Reply:
@uturuncoglu - good point. I just added it.